design/tracker_queries.go

```diff
@@ -39,6 +39,7 @@ var trackerQueryAttributes = a.Type("TrackerQueryAttributes", func() {
 var trackerQueryRelationships = a.Type("TrackerQueryRelations", func() {
 	a.Attribute("tracker", relationKindUUID, "This defines the related tracker")
 	a.Attribute("space", relationSpaces, "This defines the owning space")
+	a.Attribute("baseType", relationBaseType, "This defines type of Work Item")
```

Comment:
Why not `workItemType` instead of `baseType`? Are we following any convention here?

Reply:
`workitems` uses `baseType` with `relationBaseType` data type  -> https://github.com/fabric8-services/fabric8-wit/blob/master/design/workitems.go#L40

Reply:
@kwk @baijum done here 
4e28161